llvm/test/CodeGen/AArch64/llvm.sincos.ll

```diff
@@ -0,0 +1,516 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 2
+; RUN: llc -mtriple=aarch64-gnu-linux < %s | FileCheck -check-prefixes=CHECK %s
```

Comment:
Are there aarch64/arm targets that don't have sincos that we can test on as well?

Reply:
I've added tests where the libcall is unavailable (and so the intrinsic is expanded to individual sin/cos calls) :+1: